-inputs.tf

```diff
@@ -159,4 +159,20 @@ variable "source_control_commit_paths" {
       path2 = "commits"
     }
   }
-}
\ No newline at end of file
+}
+
+variable "slack_notification_for_approval" {
+  type        = bool
+  default     = false
+  description = "When true - AWS chatbot service is created along with a lambda function to approve codepipeline manual approval stage"
+}
+
+variable "slack_workspace_id" {
+  description = "The workspace ID for slack account to be used for notifications"
+  type        = string
```

Comment:
Can we get a default value, maybe an empty string?  I think this would prevent this from being a breaking change.

Reply:
Agreed, this module already has so many breaking changes it's pretty unreal.